dev/com.ibm.ws.microprofile.faulttolerance.cdi/src/com/ibm/ws/microprofile/faulttolerance/cdi/FaultToleranceInterceptor.java

```diff
@@ -111,8 +111,9 @@ private AggregatedFTPolicy processPolicies(InvocationContext context, BeanManage
         Class<?> targetClass = context.getTarget().getClass();
         Annotation[] annotations = targetClass.getAnnotations();
         for (Annotation annotation : annotations) {
-            // Don't process any annotations which aren't enabled
-            if (!FTGlobalConfig.getActiveAnnotations(targetClass).contains(annotation.annotationType())) {
+
+            //Check that the annotation has not been disabled for this specific class. 
+            if (! FTGlobalConfig.isAnnotationEnabled(annotation, targetClass)) { 
```

Comment:
I don't really like this continue ... why not remove the negation and just put the whole thing inside the if block?